javascript/code-challenges/graph/breadthFirst/breadth-first.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const graph = require('../graph');
+
+breadthFirst(startNode) {
+  const queue =[];
+
+  const visitedNodes = new Set();
+
+  queue.push(startNode);
+  visitedNodes.add(startNode);
+
+  while(queue.length) {
+    const currentNode = queue.shift();
+    const neighbors = this.getNeighbor(currentNode);
+
+    for(let neighbor of neighbors) {
+      const neighborNode = neighbor.vertex;
+
+      if(visitedNodes.has(neighborNode)) {
+        continue;
+      } else {
+        visitedNodes.add(neighborNode);
+      }
+      queue.push(neighborNode);
+    }
+  }
+  return visitedNodes;
+}
+
+export mo
```

Comment:
Code is looking good Tahmina! Once you have a chance to come back to this and add tests I will regrade for you!